examples/spec/twilio_api_v2010.yaml

```diff
@@ -432,6 +435,62 @@ paths:
                   $ref: '#/components/examples/test.response_object.example'
       security:
         - accountSid_authToken: [ ]
+    post:
+      operationId: UpdateCall
+      parameters:
+        - in: path
+          name: AccountSid
+          required: true
+          schema:
+            maxLength: 34
+            minLength: 34
+            pattern: ^AC[0-9a-fA-F]{32}$
+            type: string
+        - description: INTEGER ID param!!!
+          in: path
+          name: TestInteger
+          required: true
+          schema:
+            type: integer
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              properties:
+                RequiredStringProperty:
+                  type: string
+                TestUri:
+                  format: uri
+                  type: string
+                TestMethod:
+                  type: string
+                  format: http-method
```

Comment:
If this is the only change required to test the fix, recommend this property be added to an existing operation to reduce the doc size.